lms/djangoapps/courseware/tests/helpers.py

```diff
@@ -22,6 +32,72 @@ def get_request_for_user(user):
     return request
 
 
+def answer_entrance_exam_problem(course, request, problem, user=None):
```

Comment:
Was talking to @mattdrayer and we thought that these should be considered helper functions since we are using them from outside the module.